service/status-report/main.go

```diff
@@ -146,7 +135,7 @@ func main() {
 		SetPlatformVersion(pltVersion).
 		SetOperatorVersion(oprVersion).
 		SetClusterID(string(clusterID)).
-		SetNamespacedName(namespacedName.String())
+		SetNamespacedName(storageClientName)
```

Comment:
Is that an upcoming change to rename the API correctly?

Reply:
Yes, #134

Reply:
ok